From cf9662beb414978d8b7424cd8fa5bffeb6aacab3 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 09 Nov 2021 16:41:18 +0200
Subject: [PATCH] [SW-0] greco: pll: add post divide power down bit

The "Post divide power down" bit was not used as a part of the
configuration sequence.
According to the PLL spec, this bit must be set, so the post divisors
are powered.
Add the FOUTPOSTDIVEN bit to the reset sequence.

Change-Id: I97644c674320fce59295d95d3a2d88fa2e931d7c
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/greco/arc/pll.c b/arch/greco/arc/pll.c
index e88522b..8699ebd 100644
--- a/arch/greco/arc/pll.c
+++ b/arch/greco/arc/pll.c
@@ -51,7 +51,7 @@
 }
 
 #define GRECO_PLL_CFG		0x100
-#define GRECO_PLL_CFG_EN	BIT(0)
+#define GRECO_PLL_CFG_EN	(BIT(0) | BIT(8))
 #define GRECO_PLL_CFG_LOCK	BIT(16)
 #define GRECO_PLL_DIV_SEL_DELAY	0x1E0
 
